### DISCUSSION TITLE
glove.42B.300d.txt is missing

### DISCUSSION THREAD

**Discussion by e8606876** [sebastian-hofstaetter/teaching#71] [Advanced IR: Exercise Q&A] [answered]
The dataset provided for download seems to be incomplete.

Should we download the file from kaggle?

https://www.kaggle.com/datasets/yutanakamura/glove42b300dtxt

**Comment by bruluk5w** [comment, marked as answer]
The exercise statement lists a link where you can download the data.
It's: http://nlp.stanford.edu/data/glove.42B.300d.zip

**Reply by LuisKolb** [reply]
that particular link is broken, only works with https, like https://nlp.stanford.edu/data/glove.42B.300d.zip  
edit: the file downloaded from there seems to be corrupted

**Reply by maximilianvie** [reply]
The link (https://nlp.stanford.edu/data/glove.42B.300d.zip) doesn't unfortunately doesn't work. Where can we download the glove embedding?

**Reply by whoopi24** [reply]
Hi,
I think we downloaded it manually from https://nlp.stanford.edu/projects/glove/, but it says now that this is not a secure connection, so I was not able to check.
Best regards, 
Marina

**Reply by dblasko** [reply]
Hi, 
in case the Stanford link above doesn't work for you (it did not for me), the embeddings are available here: [https://huggingface.co/stanfordnlp/glove/resolve/main/glove.42B.300d.zip](https://huggingface.co/stanfordnlp/glove/resolve/main/glove.42B.300d.zip)
